core/block_svg.js

```diff
@@ -934,14 +935,14 @@ Blockly.BlockSvg.prototype.dispose = function(healStack, animate) {
 
   // Clear pending warnings.
   if (this.warningTextDb_) {
-    for (var n in this.warningTextDb_) {
+    for (const n in this.warningTextDb_) {
```

Comment:
[5.8.1](https://google.github.io/styleguide/jsguide.html#features-for-loops) Prefer `Object.keys` of for in.

Reply:
Done